telegram/_chatfullinfo.py

```diff
@@ -19,58 +19,375 @@
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 """This module contains an object that represents a Telegram ChatFullInfo."""
 from datetime import datetime
-from typing import TYPE_CHECKING, Optional, Sequence
+from typing import TYPE_CHECKING, Optional, Sequence, Tuple
 
 from telegram._birthdate import Birthdate
-from telegram._chat import Chat
+from telegram._chat import Chat, _ChatBase
 from telegram._chatlocation import ChatLocation
 from telegram._chatpermissions import ChatPermissions
 from telegram._files.chatphoto import ChatPhoto
 from telegram._reaction import ReactionType
+from telegram._utils.argumentparsing import parse_sequence_arg
+from telegram._utils.datetime import extract_tzinfo_from_defaults, from_timestamp
 from telegram._utils.types import JSONDict
 
 if TYPE_CHECKING:
-    from telegram import BusinessIntro, BusinessLocation, BusinessOpeningHours, Message
+    from telegram import Bot, BusinessIntro, BusinessLocation, BusinessOpeningHours, Message
 
 
-class ChatFullInfo(Chat):
+class ChatFullInfo(_ChatBase):
     """
     This object contains full information about a chat.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
     considered equal, if their :attr:`~telegram.Chat.id` is equal.
 
-    Caution:
```

Comment:
maybe add a versionchanged for this?

Reply:
what should I write? that you can access Chat's shortcuts as well?